client/src/components/pages/Home/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Button from '../../common/Button';
+import './index.css';
+
+const HomePage = () => {
+  return (
+    <div className="home">
+      <div className="home__header">
+        <div className="home__header__img__overlay"></div>
+      </div>
+      <div className="about">
+        <h3 className="about__title">About</h3>
+        <div className="about__block">
+          <div className="about__block__text">
+            <p>
+              Lorem ipsum is placeholder text commonly used in the graphic,
+              print, and publishing industries for previewing layouts and visual
+              mockups.
+            </p>
+          </div>
+        </div>
+      </div>
+      <div className="buttons">
```

Comment:
`buttons` is very generic and doesn't say much, how bout naming it `account` and then the below classes as `account__login` and `account__signup`?

Reply:
Done